mixer/pkg/server/args.go

```diff
@@ -124,7 +124,7 @@ func DefaultArgs() *Args {
 		ReadinessProbeOptions:  &probe.Options{},
 		IntrospectionOptions:   ctrlz.DefaultOptions(),
 		EnableProfiling:        true,
-		NumCheckCacheEntries:   5000 * 5 * 60, // 5000 QPS with average TTL of 5 minutes
+		NumCheckCacheEntries:   0, // prior value 5000 * 5 * 60, 5000 QPS with average TTL of 5 minutes
```

Comment:
I think you should this as is.

Instead, set the value to 0 explicitly right before creating the cache. We want it to be disabled no matter what the user says on the command-line.